examples/templates/exposed-thing/package.json

```diff
@@ -16,10 +16,10 @@
         "tslint": "5.12.1"
     },
     "dependencies": {
-        "@node-wot/binding-http": "0.8.3",
-        "@node-wot/core": "0.8.3",
+        "@node-wot/binding-http": "^0.8.2",
+        "@node-wot/core": "^0.8.2",
         "request": "2.88.0",
-        "ajv": "^7.0.4"
+        "ajv": "^6.12.6"
```

Comment:
@egekorkan May I ask your input since you were maintaining the templates

Reply:
I think that this is fine. We should however upgrade the actual code that uses ajv at some point.

Reply:
@ArPhil I also like the .x notation, good idea!